[2023-01-29T20:21:23.167+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_pipeline_3.load_data manual__2023-01-29T20:20:12.705450+00:00 [queued]>
[2023-01-29T20:21:23.188+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_pipeline_3.load_data manual__2023-01-29T20:20:12.705450+00:00 [queued]>
[2023-01-29T20:21:23.188+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-29T20:21:23.189+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 3
[2023-01-29T20:21:23.189+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-29T20:21:23.220+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): load_data> on 2023-01-29 20:20:12.705450+00:00
[2023-01-29T20:21:23.234+0000] {standard_task_runner.py:55} INFO - Started process 1150 to run task
[2023-01-29T20:21:23.246+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_3', 'load_data', 'manual__2023-01-29T20:20:12.705450+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpr73s5xgb']
[2023-01-29T20:21:23.250+0000] {standard_task_runner.py:83} INFO - Job 12: Subtask load_data
[2023-01-29T20:21:23.312+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-01-29T20:21:23.532+0000] {task_command.py:388} INFO - Running <TaskInstance: data_pipeline_3.load_data manual__2023-01-29T20:20:12.705450+00:00 [running]> on host 84fb6dd1372f
[2023-01-29T20:21:23.707+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=abdessamad
AIRFLOW_CTX_DAG_ID=data_pipeline_3
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2023-01-29T20:20:12.705450+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-29T20:20:12.705450+00:00
[2023-01-29T20:21:23.814+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-29T20:21:23.832+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=data_pipeline_3, task_id=load_data, execution_date=20230129T202012, start_date=20230129T202123, end_date=20230129T202123
[2023-01-29T20:21:23.895+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-29T20:21:23.932+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
